2023-02-11 00:53:38,349 INFO    MainThread:6812 [wandb_setup.py:_flush():68] Configure stats pid to 6812
2023-02-11 00:53:38,350 INFO    MainThread:6812 [wandb_setup.py:_flush():68] Loading settings from C:\Users\GCM\.config\wandb\settings
2023-02-11 00:53:38,350 INFO    MainThread:6812 [wandb_setup.py:_flush():68] Loading settings from E:\GIT_REPOS\LAB\Beetle_classifier\Train\wandb\settings
2023-02-11 00:53:38,350 INFO    MainThread:6812 [wandb_setup.py:_flush():68] Loading settings from environment variables: {'_require_service': 'True'}
2023-02-11 00:53:38,350 INFO    MainThread:6812 [wandb_setup.py:_flush():68] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-02-11 00:53:38,350 INFO    MainThread:6812 [wandb_init.py:_log_setup():492] Logging user logs to .\wandb\run-20230211_005338-p53vefdn\logs\debug.log
2023-02-11 00:53:38,351 INFO    MainThread:6812 [wandb_init.py:_log_setup():493] Logging internal logs to .\wandb\run-20230211_005338-p53vefdn\logs\debug-internal.log
2023-02-11 00:53:38,351 INFO    MainThread:6812 [wandb_init.py:_jupyter_setup():438] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x0000029A53EAFBB0>
2023-02-11 00:53:38,351 INFO    MainThread:6812 [wandb_init.py:init():532] calling init triggers
2023-02-11 00:53:38,351 INFO    MainThread:6812 [wandb_init.py:init():538] wandb.init called with sweep_config: {}
config: {}
2023-02-11 00:53:38,351 INFO    MainThread:6812 [wandb_init.py:init():588] starting backend
2023-02-11 00:53:38,351 INFO    MainThread:6812 [wandb_init.py:init():592] setting up manager
2023-02-11 00:53:38,365 INFO    MainThread:6812 [backend.py:_multiprocessing_setup():106] multiprocessing start_methods=spawn, using: spawn
2023-02-11 00:53:38,367 INFO    MainThread:6812 [wandb_init.py:init():599] backend started and connected
2023-02-11 00:53:38,376 INFO    MainThread:6812 [wandb_run.py:_label_probe_notebook():1203] probe notebook
2023-02-11 00:53:38,377 INFO    MainThread:6812 [wandb_run.py:_label_probe_notebook():1213] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2023-02-11 00:53:38,377 INFO    MainThread:6812 [wandb_init.py:init():687] updated telemetry
2023-02-11 00:53:38,425 INFO    MainThread:6812 [wandb_init.py:init():727] communicating run to backend with 60.0 second timeout
2023-02-11 00:54:08,440 WARNING MsgRouterThr:6812 [router.py:message_loop():77] message_loop has been closed
2023-02-11 00:54:08,463 WARNING MsgRouterThr:6812 [router.py:message_loop():77] message_loop has been closed
2023-02-11 00:54:08,556 WARNING MainThread:6812 [wandb_init.py:init():1154] interrupted
Traceback (most recent call last):
  File "C:\Users\GCM\anaconda3\envs\BC_310\lib\site-packages\wandb\sdk\wandb_init.py", line 1133, in init
    run = wi.init()
  File "C:\Users\GCM\anaconda3\envs\BC_310\lib\site-packages\wandb\sdk\wandb_init.py", line 730, in init
    result = run_init_handle.wait(
  File "C:\Users\GCM\anaconda3\envs\BC_310\lib\site-packages\wandb\sdk\lib\mailbox.py", line 273, in wait
    found, abandoned = self._slot._get_and_clear(timeout=wait_timeout)
  File "C:\Users\GCM\anaconda3\envs\BC_310\lib\site-packages\wandb\sdk\lib\mailbox.py", line 120, in _get_and_clear
    if self._wait(timeout=timeout):
  File "C:\Users\GCM\anaconda3\envs\BC_310\lib\site-packages\wandb\sdk\lib\mailbox.py", line 116, in _wait
    return self._event.wait(timeout=timeout)
  File "C:\Users\GCM\anaconda3\envs\BC_310\lib\threading.py", line 607, in wait
    signaled = self._cond.wait(timeout)
  File "C:\Users\GCM\anaconda3\envs\BC_310\lib\threading.py", line 324, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
